doc/manual_experimental.rst

```diff
@@ -434,6 +434,32 @@ This operator will be matched against assignments to missing fields.
 .. code-block:: nim
   a.b = c # becomes `.=`(a, b, c)
 
+Call operator
+-------------
+The call operator, `()`, matches all kinds of unresolved calls, however
+it does not match missing proc overloads. It must be enabled with
+``{.experimental: "callOperator".}`` to use.
+
+.. code-block:: nim
+  block:
+    let a = 1
+    let b = 2
+    a.b # becomes `()`(a, b)
```

Comment:
this is incorrect and works "by chance" if (a,b) == (b,a)
use this instead, and also fix the reminaing examples + tests
```suggestion
  template `()`(a: int, b: float): untyped = $(a, b)
  block:
    let a = 1.0
    let b = 2
    {.push experimental: "callOperator".}
    doAssert a.b == `()`(b, a)
    {.pop.}
    doAssert not compiles(a.b)
```